suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -734,6 +734,7 @@ export const isAccountOutdated = (account: Account, freshInfo: AccountInfo) => {
             return (
                 freshInfo.misc!.nonce !== account.misc.nonce ||
                 freshInfo.balance !== account.balance || // balance can change because of beacon chain txs (staking) |
+                freshInfo.tokens !== account.tokens ||
                 JSON.stringify(freshInfo?.misc?.stakingPools) !==
                     JSON.stringify(account?.misc?.stakingPools)
```

Comment:
probably it would be better to compare string instead of object, maybe that's why it sometimes work and sometimes not.

Please inspired by lines for staking